docs/source/examples/pec_tutorial.myst

```diff
@@ -0,0 +1,308 @@
+
+In PEC, the first step is to characterize the set of noisy, implementable operations $\{\mathcal{O_{\alpha}}\}$ of a quantum device so that we can represent the ideal (noiseless) operations $\{\mathcal{G}_i\}$ of a circuit in this basis, namely
+
+$$
+\mathcal{G}_i = \sum_{\alpha} \eta_{i, \alpha} \mathcal{O}_{\alpha}.
+$$
+
+Note that the calligraphic symbols $\mathcal{G}_i$ and $\mathcal{O}_{\alpha}$ stand for super-operators acting on the quantum state of the qubits as linear quantum channels, and $\eta_{i, \alpha} \in \mathbb{R}$.
+
+## Setup
+
+We begin by importing the relevant modules and libraries that we will require
+for the rest of this tutorial.
+
+```{code-cell} ipython3
+import functools
+from typing import List, Tuple, Union
+
+# Plotting imports.
+import matplotlib.pyplot as plt
+plt.rcParams.update({"font.family": "serif", "font.size": 15})
+%matplotlib inline
+
+# Third-party imports.
+import cirq
+import qiskit
+
+import networkx as nx
+import numpy as np
+
+from braket.devices import LocalSimulator
+from braket.circuits import Circuit, gates, Noise
+
+# Mitiq imports.
+from mitiq import benchmarks, pec
+
+import warnings
+warnings.filterwarnings('ignore')
+```
+
+## Define parameters
+
+```{code-cell} ipython3
+# Random seed for circuit generation.
+seed: int = 1               
+
+# Total number of shots to use.
+# For PEC, the shots per circuit is shots / num_samples.
+shots: int = 10_000     
+
+# Number of samples (circuits) to use in PEC.    
+num_samples: int = 100  
+    
+# Qubits to use on the experiment. 
+qubits: List[int] = [0, 1, 2]
+        
+# Average results over this many trials (circuit instances) at each depth.
+trials: int = 4      
+    
+# As operating on simulator, assume a constant CNOT error rate.
+cnot_error_prob: float = 0.01
+
+# Clifford depths.
+depths: List[int] = [1, 5, 9]
+```
+
+Next, we need to define both the noisy and ideal simulator backends that we will
+run our experiments on. In this case, we will be making use of the
+[`braket_dm`](https://docs.aws.amazon.com/braket/latest/developerguide/braket-send-to-local-simulator.html) simulator. 
+
+We also define a graph representation of our qubits and assume a line topology.
+
+```{code-cell} ipython3
+# Assume chain-like connectivity
+computer = nx.Graph()
+computer.add_edges_from([(0, 1), (1, 2)])
+
+# Add reversed edges to computer graph.
+# This is important to represent CNOT gates with target and control reversed.
+computer = nx.to_directed(computer)
+
+noisy_backend = LocalSimulator("braket_dm")
+ideal_backend = LocalSimulator("default")
+```
+
+## Define the circuit
+
+We use mirror circuits to benchmark the performance of the device. Mirror circuits, introduced in [arXiv:2008.11294](https://arxiv.org/abs/2008.11294), are designed such that only one bitstring should be sampled. When run on a device, any other measured bitstrings are due to noise. The frequency of the correct bitstring is our target metric.
```

Comment:
Change the arxiv link with reference showing up in the bibliography of Mitiq or add reference after the arxiv link.